app/views/categories/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="flex flex-col h-screen justify-between">
+  <div>
+    <div class="navigation py-6 px-3 blue-nav-white-text">
+      <ul class="flex justify-between items-center">
+        <li class="w-10"><%= link_to image_tag('backarrow.png', alt: 'Back'), categories_path, class: 'nav-link' %></li>
+        <h2><strong><%= @category.name %></strong></h2>
+        <li><%= link_to 'Sign Out', destroy_user_session_path, data: { turbo_method: :delete }, class: "nav-link" %></li>
+      </ul>
+    </div>
+    <p style="color: green"><%= notice %></p>
+    <%= render @category %>
+    <section class="px-5">
+      <div id="purchases">
+        <% @purchases.each do |purchase| %>
+          <a href="<%= category_purchase_path(@category, purchase) %>" class="m-1">
+            <%= render "purchases/purchase", purchase: purchase %>
+          </a>
+        <% end %>
+      </div>
+    </section>
+  </div>
+  <section class="flex flex-col space-y-2 p-5">
+    <a href="<%= new_category_purchase_path(@category) %>" class="">
+      <button data-testid="new-purchase-button" class="drop-shadow-md p-3 w-full green-btn text-white rounded-md hover:bg-green-600 focus:outline-none focus:bg-green-600">
+        New purchase
+      </button>
+    </a>
+    <%= link_to edit_category_path(@category) do %>
+      <button data-testid="edit-category-button" class="drop-shadow-md p-3 orange-btn bg-white w-full rounded hover:bg-orange-600">Edit category</button>
+    <% end %>
+    <%= button_to "Delete category", @category, method: :delete, data: { confirm: 'Are you sure?' }, class: "drop-shadow-md w-full p-3 red-btn text-white rounded-md hover:bg-red-600 focus:outline-none focus:bg-red-600" %>
+  </section>
```

Comment:
- [ ] From the design the only button that should be displayed at the button of the purchases list page should be the new transaction button, the other two should be removed. Kindly do that.

- [ ] Kindly style the new purchase/transaction button to fit the design, just as you have done with the new category button earlier

  ![image](https://github.com/aradradev/Budget-app/assets/54848363/de9d437f-0a48-4b02-98ab-8b6b748673ba)